.github/workflows/intermittent-test-check.yml

```diff
@@ -145,12 +190,6 @@ jobs:
         with:
           name: result-${{ env.TEST_CLASS }}-split-${{ matrix.split }}
           path: target/${{ needs.prepare-job.outputs.test_type }}
-      - name: Delete temporary build artifacts before caching
```

Comment:
Is this part removed because of the restore action handling this?

Reply:
Yes, when using `cache/restore`, it will not save the cache when the job is done.

Reply:
Thanks for explaining.

Reply:
See HDDS-10028 for related change in regular CI.